### PR TITLE
FIX CSS :)

### PR DESCRIPTION
**Pull request by Ragbecca**

### CODE REVIEW

**Comment by CryptForge** [comment]
oh dear god

File: website/src/component/AuthButton/LogoutButton.jsx [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
   const auth = useContext(AuthContext);
+  const nav = useNavigate();
+
+  const logout = () => {
+    auth.userLogout();
+    nav("/login");
+  };
 
   return (
-    <div className="navigation-link">
```
Comment (vschonk) [inline website/src/component/AuthButton/LogoutButton.jsx:11]: Where did the popup go?

**Review by vschonk — CHANGES_REQUESTED** [review]
My popup is gone :(

**Review by vschonk — APPROVED** [review]
cloned code, it works, no problems

**Pull request merged**